dashboard/modules/runtime_env/runtime_env_agent.py

```diff
@@ -118,11 +260,16 @@ def get_or_create_logger(self, job_id: bytes):
             self._per_job_logger_cache[job_id] = per_job_logger
         return self._per_job_logger_cache[job_id]
 
-    async def CreateRuntimeEnv(self, request, context):
+    async def CreateRuntimeEnvOrGet(self, request, context):
```

Comment:
Any reason why it is not `GetOrCreateRuntimeEnv`?

Reply:
Just make `CreateRuntimeEnv...` and `DeleteRuntimeEnv...` symmetrical. 😅

Reply:
Rename done!